discord/appinfo.py

```diff
@@ -52,9 +55,28 @@ class AppInfo:
         grant flow to join.
     rpc_origins: Optional[List[:class:`str`]]
         A list of RPC origin URLs, if RPC is enabled.
+    summary: Optional[:class:`str`]
+        If this application is a game sold on Discord,
+        this field will be the summary field for the store page of its primary SKU
+    verify_key: Optional[:class:`str`]
```

Comment:
```suggestion
    verify_key: :class:`str`
```

This field is also always there.